lib/addons/karpenter/index.ts

```diff
@@ -87,6 +87,17 @@ interface KarpenterAddOnProps extends HelmAddOnUserProps {
      * Only applicable for v0.19.0 and later
      */
     interruptionHandling?: boolean,
+
+    /**
+     * Limits define a set of bounds for provisioning capacity.
```

Comment:
Do you mind adding the following to the doc (it will be published under the api reference: https://aws-quickstart.github.io/cdk-eks-blueprints/api/): 
```
Resource limits constrain the total size of the cluster.
Limits prevent Karpenter from creating new instances once the limit is exceeded.
```